src/pseudopeople/utilities.py

```diff
@@ -30,11 +31,83 @@ def get_configuration(user_yaml_path: Union[Path, str] = None) -> ConfigTree:
         data=Path(pseudopeople.__file__).resolve().parent / "default_configuration.yaml",
         layers=default_config_layers,
     )
-    if user_yaml_path:
-        noising_configuration.update(user_yaml_path, layer="user")
+    if user_config:
+        if isinstance(user_config, (Path, str)):
+            with open(user_config, "r") as f:
+                # TODO: Do we need to support other filetypes that yaml?
```

Comment:
do we need to support other filetypes than yaml?

Reply:
I don't think so.